docs/man/grcli-conntrack.1.md

```
@@ -0,0 +1,38 @@
+GRCLI-CONNTRACK 1 @DATE@ "grout @VERSION@"
+===========================================
+
+# NAME
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)